See http://sources.redhat.com/ml/libc-alpha/2003-12/msg00101.html
Fixes
../sysdeps/unix/sysv/linux/powerpc/sys/procfs.h:58: error: conflicting types  for 'elf_vrreg_t' 
when building glibc-2.3.2 for powerpc64 and linux-2.6.x

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/unix/sysv/linux/powerpc/sys/procfs.h,v
retrieving revision 1.10
retrieving revision 1.11
diff -u -r1.10 -r1.11
--- libc/sysdeps/unix/sysv/linux/powerpc/sys/procfs.h	2003/01/12 08:24:23	1.10
+++ libc/sysdeps/unix/sysv/linux/powerpc/sys/procfs.h	2003/12/17 23:09:34	1.11
@@ -45,7 +45,6 @@
 
 typedef double elf_fpreg_t;
 typedef elf_fpreg_t elf_fpregset_t[ELF_NFPREG];
-#endif
 
 /* gcc 3.1 and newer support __uint128_t.  */
 #if !__GNUC_PREREQ(3,1)
@@ -57,6 +56,7 @@
 /* Altivec registers */
 typedef __uint128_t elf_vrreg_t;
 typedef elf_vrreg_t elf_vrregset_t[ELF_NVRREG];
+#endif
 
 struct elf_siginfo
   {
